libraries/ml/src/viewers/activity-cliffs.ts

```diff
@@ -70,7 +69,7 @@ export async function getActivityCliffs(df: DG.DataFrame, seqCol: DG.Column, enc
   methodName: string, semType: string, tags: {[index: string]: string},
   seqSpaceFunc: (params: ISequenceSpaceParams) => Promise<ISequenceSpaceResult>,
   simMatrixFunc: (dim: number, seqCol: DG.Column, df: DG.DataFrame, colName: string,
-    simArr: DG.Column[]) => Promise<DG.Column[]>,
+    simArr: (DG.Column | null)[]) => Promise<(DG.Column | null)[]>,
```

Comment:
Here the `DG.Column` implicitly has type `any`: `DG.Column<any>`.
Does column type matter here and in the other places through the code or the column can contain any data?